contracts/src/protocol/IPTokenStaking.sol

```diff
@@ -31,7 +31,7 @@ contract IPTokenStaking is IIPTokenStaking, Ownable2StepUpgradeable, ReentrancyG
     /// @notice Stake amount increments, 1 ether => e.g. 1 ether, 2 ether, 5 ether etc.
     uint256 public immutable STAKE_ROUNDING;
 
-    /// @notice Default minimum unjail fee
+    /// @notice Default minimum validator update fee
     uint256 public immutable DEFAULT_MIN_UNJAIL_FEE;
```

Comment:
need to be changed as well?

Reply:
Refactoring to "fee" so it applies to operator methods as well